plugins/Toolbox/src/SubscribedPackagesModel.py

```diff
@@ -27,20 +28,34 @@ def addValue(self, discrepancy):
             self._discrepancies = discrepancy
 
     def update(self):
-        items = []
+        self._items.clear()
 
         for item in self._metadata:
             if item["package_id"] not in self._discrepancies:
                 continue
             package = {"name": item["display_name"], "sdk_versions": item["sdk_versions"]}
             if self._sdk_version not in item["sdk_versions"]:
-                package.update({"is_compatible": "False"})
+                package.update({"is_compatible": False})
             else:
-                package.update({"is_compatible": "True"})
+                package.update({"is_compatible": True})
             try:
                 package.update({"icon_url": item["icon_url"]})
             except KeyError:  # There is no 'icon_url" in the response payload for this package
                 package.update({"icon_url": ""})
 
-            items.append(package)
-        self.setItems(items)
\ No newline at end of file
+            self._items.append(package)
+        self.setItems(self._items)
+
+    def has_compatible_packages(self):
```

Comment:
It's not a property, so it should be hasCompatiblePackages (functions are lower camel case)